api/audit-trail-gw/src/middlewares/metrics.ts

```diff
@@ -0,0 +1,63 @@
+import promClient, { Histogram, Gauge, Registry } from 'prom-client';
```

Comment:
if this is the same as for the ssi-bridge we should move it to is-shared-components